### ISSUE TITLE
poor results on Pascal Person Part of gnn_parse

### ISSUE THREAD

**Issue by jianzhang96** [hlzhu09/Hierarchical-Human-Parsing#12]
Thanks for your excellent work.
Recently I want reproduce this work. I download the ImageNet pretrained model *resnet101-imagenet.pth* and find it can be loaded. I trained the *baseline* and got 69.16 mIoU (70.51 with multi-scale test). Then, I trained *gnn_parse* with *gnn_loss_att* and only got 69.01 mIoU (70.21 with multi-scale test). The batch size is set to 20 and learning rate is set to 7e-3.
Is there something wrong with my reproduction?

**Comment by dandanAD** [comment]
Is this file resnet101_stem.pth using resnet101-imagenet.pth

**Comment by Minssnail** [comment]
收到啦！我会尽快回复您。
